### ISSUE TITLE
Failure to send messages to discord when token image has spaces in its filename. 

### ISSUE THREAD

**Issue by Opius134** [therealguy90/foundrytodiscord#33]
This bug is present in Foundry Version 11 (11.315) while running in the DnD5e Game System (3.0.4)

**To Reproduce**
Select a token with a space in its filename and attempt to send a message.

The chat message appears in Foundry without issue, but it fails to get to Discord. Console log spits out a 404 error. 

**Expected behavior**
The resulting message should mirror from Foundry chat to the linked Discord channel.

**Screenshots**
![image](https://github.com/therealguy90/foundrytodiscord/assets/100728089/73f1f5b1-c936-4cf5-a1bd-4c648aab4330)
![image](https://github.com/therealguy90/foundrytodiscord/assets/100728089/877fcd7a-48be-4613-bf6d-1c914103ef6f)

**Additional context**
After testing, I think this issue came up by using Aedif's Token Variant Art during token configuration. Selecting a file through the normal file browser appends a '%20' on spaces in filenames, whereas going through the token variant retains the filename space.

**Comment by therealguy90** [comment]
Yeah, this is because back in v9, filepaths on Foundry weren't using an encoded URI scheme, which is probably retained in that module. There are two ways to go about this: opening an issue on Aedif's Token Variant to make the filepaths properly encoded, or I can just fix it myself here.

**Comment by therealguy90** [comment]
Should be fixed in c4ca4de476f64bb2422827cc724f64743e7b7d1e of the 2.0 branch. Unfortunately, Foundry's FilePicker doesn't have a native way to encode file paths, so I had to do it manually through crawling. Other special characters should be included too, not just spaces.

**Comment by therealguy90** [comment]
Fixed in 2.0.0.
